bridges/snowbridge/primitives/router/src/inbound/mod.rs

```diff
@@ -131,8 +131,6 @@ pub enum ConvertMessageError {
 	UnsupportedVersion,
 	InvalidDestination,
 	InvalidToken,
-	/// The fee asset is not supported for conversion.
```

Comment:
No idea where this is used. However, this changes the encoding. Are you aware of this?

Reply:
@bkchr it is not used anywhere but you make a good point. I don't think removing this adds any benefit really. I could do:

```
pub enum ConvertMessageError {
    /// The message version is not supported for conversion.
    UnsupportedVersion = 0,
    InvalidDestination = 1,
    InvalidToken = 2,
    /// Keeping the same index as before.
    CannotReanchor = 4,
}
```

It's ugly, I'd rather just leave it in. @alistair-singh @yrong @vgeddes I'm going to close this PR unless you have reservations.